### PR TITLE
`nupac -v`/`--version`

### PR DESCRIPTION
**Pull request by skelly37**

### CODE REVIEW

**Comment by Yethal** [comment]
```# Nushell package manager
export def "nupac" [
    --version(-v): bool # Display nupac version
    --help(-h): bool # Display this help message
] {
    if $version {
        get-metadata (get-package-location (get-packages 'nupac'|get 0))
    } else {
        nupac --help
    }
}```
While this may seem like an overengineered solution this does allow us to accomplish the same thing without duplicating a variable while also serving as a simple test of nupac's internal functions

File: modules/nupac.nu [hunk 1]
```diff
@@ -11,11 +11,6 @@
 #?keywords: [package, management]
 #?os: [android, linux, macos, windows]
```
Comment (skelly37) [inline modules/nupac.nu:13]: Please bring it back. I thought we agreed on using verbose, auto-generated `nupac version`, while using the hardcoded one in `export def nupac`

```suggestion

# nupac's version
def VERSION [] {
    "0.1.0"
}
```

File: modules/nupac.nu [hunk 2]
```diff
@@ -313,7 +308,7 @@ export def "nupac" [
     --help(-h): bool # Display this help message
 ] {
     if $version {
-        nupac version
+        nupac version|get version
```
Comment (skelly37) [inline modules/nupac.nu:311]:
```suggestion
        VERSION
```

File: modules/nupac.nu (hunk 1 above)
Comment (Yethal) [reply modules/nupac.nu]: There is really no value in displaying a hardcoded version that's an exact duplicate of the same version we store few lines above

Comment (skelly37) [reply modules/nupac.nu]: If you insist, metadata will be moved to a separate file tho

**Review by Yethal — APPROVED** [review]

**Review by Yethal — APPROVED** [review]

**Pull request merged**